admin_guide/out_of_resource_handling.adoc

```diff
@@ -64,8 +68,7 @@ Each threshold is of the following form:
 <eviction_signal><operator><quantity>
 ----
 
-* Valid `eviction-signal` tokens as defined in
-xref:out-of-resource-eviction-signals[Eviction Signals].
+* Valid `eviction-signal` tokens as defined by eviction signals.
 * Valid `operator` tokens are `<`.
```

Comment:
right now, < is the only supported operator, but there was discussion on potentially offering more.  i kept the doc this way to allow it to grow without major re-writes.